demo/bike/main.js

```diff
@@ -1,13 +1,17 @@
-const {KalmanFilter} = kalmanFilter;// eslint-disable-line no-undef
-const createElement = require('../shared/views/create-element');
+/**
+ * @typedef {typeof import('../../index.d.ts')} kalmanFilter
+ */
+const { KalmanFilter } = /** @type {kalmanFilter} */ (kalmanFilter);// eslint-disable-line no-undef
+const createElement = require('../shared/views/create-elem§ent');
 const createGroupBoxes = require('../shared/views/create-group-boxes');
 const noisyObservations = require('./observations.json').observations;
 const kfOptions = require('./kf-options');
-
 const kf = new KalmanFilter(kfOptions);
```

Comment:
here `kf` is properly typed as a KalmanFilter instance.

Reply:
ok